### ISSUE TITLE
add extension version function

### ISSUE THREAD

**Issue by caron** [fabric-engine/kludge#6]
I wanted to be able to add the version of the extension to the binding description language, so I did it locally on my libkludge that ships with 2.5. Might be something you guys want to add too?

`ext.add_ext_version_spec(1, 0, 0)`

added to ext.py...
```
def add_ext_version_spec(self, major=0, minor=0, revision=0):
    self.debug("Extension: Adding version specification number %s.%s.%s" % (major, minor, revision))
    if (major + minor + revision) > 0:
      self.ext_version = "%s.%s.%s" % (major, minor, revision)
```

modified ext.fpm.json template...
```
{% if ext.dfg_version %}
  "version": "{{ext.ext_version}}",
  {% endif %}
```

**Comment by pzion** [comment]
sounds great!  Are you able to submit a pull request?  We would be happy to merge it in

**Comment by pzion** [comment]
If that's tricky at all then I can just manually add it, just let me know

**Comment by caron** [comment]
It's not tricky, just need a free minute to fork and do the pull request.

**Comment by caron** [comment]
check out #7

**Comment by caron** [comment]
fixed in ae98fd5
